tests/foreman/ui/test_oscap_tailoringfiles.py

```diff
@@ -39,12 +47,20 @@ class TailoringFilesTestCase(UITestCase):
     @classmethod
     def setUpClass(cls):
         super(TailoringFilesTestCase, cls).setUpClass()
-        file_path = settings.oscap.tailoring_path
-        cls.tailoring_path = get_data_file(file_path)
+        cls.tailoring_path = get_data_file(settings.oscap.tailoring_path)
+        proxy = entities.SmartProxy().search(
```

Comment:
isn't it safe to assume that an internal proxy has always id == 1?

Reply:
the 1st element will always be equal to the hostname , i.e default internal capsule